src/main/python/cfn_sphere/exceptions.py

```diff
@@ -30,9 +43,11 @@ class InvalidEncryptedValueException(CfnSphereException):
     pass
 
 
-class CfnSphereBotoError(CfnSphereException):
+class CfnSphereBotoErrorException(CfnSphereException):
```

Comment:
Also, not having this change would make the pull-request significantly smaller and easier to grok.